instrumentation/spring/spring-boot-resources/library/src/test/resources/bootstrap-multi.yml

```diff
@@ -0,0 +1,8 @@
+spring:
+  application:
+    name: cat-store-bootstrap
+
```

Comment:
You need to use the `---` separator to have separate documents in a single file
```suggestion

---
```

Reply:
Fixed, thank you very much